ocp_resources/csi_storage_capacity.py

```diff
@@ -0,0 +1,67 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class CSIStorageCapacity(NamespacedResource):
+    """
+    CSIStorageCapacity object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/storage_apis/storage-apis-index.html#csistoragecapacity-storage-k8s-iov1
+    """
+
+    api_group = NamespacedResource.ApiGroup.STORAGE_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        capacity=None,
+        maximum_volume_size=None,
+        node_topology=None,
+        storage_class_name=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
+    ):
+        """
+        Args:
+            name (str): Name of the endpoints resource
+            namespace (str): Namespace of endpoints resource
+            client: (DynamicClient): DynamicClient for api calls
+            capacity (str): value reported by the csi driver
+            maximum_volume_size (str): maximum volume size reported by csi driver
+            storage_class_name (str): storage class name
+            node_topology (object): defines which node has access to the storage for which capacity was reported
```

Comment:
the type is `LabelSelector`, can you add an example?

Reply:
I will add example.